moped-editor/src/views/projects/projectsListView/ProjectListViewTest/dataProvider/useSearch.js

```diff
@@ -0,0 +1,68 @@
+import { useState, useMemo } from "react";
+import { ProjectsListViewQueryConf } from "../../ProjectsListViewQueryConf";
+
+/**
+ * Attempts to retrieve a valid graphql search value, for example when searching on an
+ * integer/float field but providing it a string, this function returns the value configured
+ * in the invalidValueDefault field in the search object, or null.
+ * @param {string} column - The name of the column to search
+ * @param {*} value - The value in question
+ * @returns {*} - The value output
+ */
+export const getSearchValue = (column, value) => {
+  // Retrieve the type of field (string, float, int, etc)
+  const type = ProjectsListViewQueryConf.columns[column].type.toLowerCase();
```

Comment:
the class name capitalization throws me off, but i do think i like these field configs.

 it probably won't ever be worth the effort, but in hindsight i feel like we could expand these field configs with additional properties for the various form input props and really cut back on places where have hardcoded field metadata.

Reply:
Yes, i think that rethinking the way some of the config works could really help out! Definitely down to rename things - it feels right to rename while we are refactoring.